conf/modules.config

```
@@ -94,6 +94,33 @@ process {
         ]
     }
 
+    withName: 'DRAGONFLYE' {
+        ext.args = {
+            if ( $meta.gsize && $meta.gsize != 'NA' ){
```

I cannot find it in the documentation right now but I thought this could be also solved with
```suggestion
            if ( !$meta.gsize?.equals('NA') ){
```
where the ? checks that it exists. But also fine as is.

---

awesome. It worked ;)